vm/src/stdlib/binascii.rs

```diff
@@ -56,6 +58,63 @@ fn binascii_unhexlify(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
     Ok(vm.ctx.new_bytes(unhex))
 }
 
+static CRC32_TABLE: [u32; 256] = [
```

Comment:
Are there any CRC32 libraries on crates.io?

Reply:
There is https://crates.io/crates/crc, but also note here https://github.com/BurntSushi/rust-snappy/blob/master/src/crc32.rs#L1 (I don't know whether it's relevant anymore, though).

Reply:
> Are there any CRC32 libraries on crates.io?

Hi adrian, I just translate the implement in cpython to rust. Would you recommend to implement by crate crc?

Reply:
@stwen77 , thank you for your contribution. As @adrian17 and @mkurnikov suggested, please have a look at this crate: https://crates.io/crates/crc . It contains the crc32 logic so that we do not need to use the crc table. This is required in CPython since C does not have such a good eco system as rust. But in rust we can use many libraries, and I think we should take advantage of this ecosystem.